evap/evaluation/models.py

```diff
@@ -445,6 +445,29 @@ class State(models.IntegerChoices):
         verbose_name=_("wait for grade upload before publishing"), default=True
     )
 
+    @property
+    def has_exam(self):
+        return self.course.evaluations.filter(name_de="Klausur", name_en="Exam").exists()
+
+    @transaction.atomic
+    def create_exam_evaluation(
+        self,
+        exam_date: date,
+    ):
```

Comment:
```suggestion
    def create_exam_evaluation(self, exam_date: date):
```

If you leave a trailing comma, our code formatter interprets that as "please always list these vertically", but we generall don't want to do that when everything fits in one line.